src/parsing/parsers/postgresql/jison/select/offset_clause.test.json

```diff
@@ -38,5 +48,23 @@
     "expectedResult": {
       "lowerCase": false
     }
+  },
+  {
+    "namePrefix": "should suggest OFFSET",
+    "beforeCursor": "SELECT * FROM testTable WHERE column = 1 ",
+    "afterCursor": "",
+    "noErrors": true,
+    "containsKeywords": ["OFFSET"],
```

Comment:
Do we want to add `LIMIT` to keywords here?

Reply:
It's will not be excess for tests 👍